### PR TITLE
Fix webgl_gpgpu_birds references error

### PR DESCRIPTION
**Pull request by gonnavis**
`references` should be computed from `birdIndex` not `triangleIndex`.

The most obvious problem of [current version](https://threejs.org/examples/?q=gpgpu#webgl_gpgpu_birds) is the diferent colors of one bird.
Second is that `WIDTH` and `trianglesPerBird` must be one odd and one even, can't both even or both odd, otherwise will produce very messy result.

By the way, also changed the left right of the wings' names by bird's self view, not camera view.

### CODE REVIEW

File: examples/webgl_gpgpu_birds.html [hunk 1]
```diff
@@ -355,14 +356,14 @@
 						0, 0, 30
 					);
 
-					// Left Wing
+					// Right Wing
```
Comment (mrdoob) [inline examples/webgl_gpgpu_birds.html:359]: I don't think this is right.

**Comment by mrdoob** [comment]
> By the way, also changed the left right of the wings' names by bird's self view, not camera view.

I find this change confusing. These names are in "world space". If you look at the bird top-view. The left wing has negative values on the x, and the right wing has positive values on the x.

**Comment by gonnavis** [comment]
Okay, I personally use to judge the left and right from the perspective of the creature itself.
Shall I roll back the change of **LEFT RIGHT** to let this pr purely target the modification of references？

**Comment by mrdoob** [comment]
> Shall I roll back the change of `LEFT RIGHT` to let this pr purely target the modification of references？

Yes please.

**Comment by gonnavis** [comment]
@mrdoob Restored.

**Comment by WestLangley** [comment]
If the bird is facing the positive-z axis, then the "right wing" points in the negative-x direction in our coordinate system.

@gonnavis is correct.

**Comment by Mugen87** [comment]
Merging. The left/right issue can be changed with another PR if necessary.

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
https://github.com/mrdoob/three.js/commit/1912046996da5a7085fa15db76a18bcfadb7bd9c 😇

**Pull request merged**